007_team_5_agent_design/007_team_5_agent_design.tex

```diff
@@ -1 +1,181 @@
+The agent stores the information gained from communication in its `memory': the information stored by this data structure is shown in Table \ref{tab:team5-memory}.
+
+\begin{table}
+    \centering
+    \begin{tabular}%
+        {| >{\raggedleft\arraybackslash}p{0.25\linewidth} | %
+        >{\raggedright\arraybackslash}p{0.65\linewidth} | %
+        }
+        \hline
+        Agent ID & An agent's unique ID\\
+        \hline
+        Agent HP & The agent's HP level\\
+        \hline
+        Food Taken & The amount of food the agent last took\\
+        % \hline
+        % Intended Food Intake & The amount of food the agent intends to take\\
+        \hline
+        Favour & The Team 5 agent's opinion of this agent\\
+        \hline
+        Days Since Last Seen & The number of days since the last message from this agent\\
+        \hline
+    \end{tabular}
+    \caption{Team 5 Agent Memory}
+    \label{tab:team5-memory}
+\end{table}
+
+This memory allows the agent to construct a social network of agents in the tower and evaluate others based on these parameters. This consequently leads to the idea of favour, a metric the Team 5 agent uses to judge other agents. If more than five days have passed since the agent's last interaction with someone, then the memory of that agent is reset. This prevents the agent from using out-of-date information in its calculation and also allows for the removal of dead agents from its memory.
+
+\subsection*{Favour}\label{sec:team5-favour}
+The favour metric quantifies the agent's opinion of others in the tower. It is bounded between 0 and 10, and in a single tick can either increase by 1, decrease by 3, or stay the same:
+
+\begin{align*}
+    \texttt{hpScoreOther} &= -1 \times \frac{\texttt{otherHP}^{1.7} \times \texttt{foodTaken}^{1.3}}{\texttt{maxHP}^3}\\
+    \texttt{hpScoreSelf} &= \frac{\texttt{ownHP}^{1.7}\times\texttt{ownAttemptFood}^{1.3}}{\texttt{maxHP}^3}\\
+    \texttt{judgement} &= 100 \times (\texttt{hpScoreOther} + \texttt{hpScoreSelf})\\
+    \texttt{unboundedFavour} &= \begin{cases}
+        \texttt{originalFavour} + 1, \quad\hfill \texttt{judgement} &> 0.075\\
+        \texttt{originalFavour} + \max{\left(\frac{\texttt{judgement}}{2}, -3\right)}, \quad\hfill\texttt{judgement} &< -2
+    \end{cases}\\
+    \texttt{favour} &= \begin{cases}
+        0, \quad\hfill \texttt{unboundedFavour} &< 0\\
+        \texttt{unboundedFavour}, \quad 0 \leq \texttt{unboundedFavour} &\leq 10\\
+        10, \quad\hfill \texttt{unboundedFavour} &> 10
+    \end{cases}\\
+\end{align*}
+
+Measuring favour allows the agent to act accordingly to how other agents are behaving around it, while taking on a passive conformist approach to decision making. If the agent views others around it more favourably, then it eats less food and aims for a lower HP in hopes of improving collective utility. The agent is also more likely to propose treaties to, and accept treaties from, highly favoured agents.
```

Comment:
```suggestion
Measuring favour allows the agent to act accordingly to how other agents are behaving around it, while taking on a passive conformist approach to decision making. If the agent views others around it more favourably, then it eats less food and aims for a lower HP in the hope of improving collective utility. The agent is also more likely to propose treaties to, and accept treaties from, highly favoured agents.
```

Reply:
I think the original was fine. Subject does X in hope(s) of achieving Y - this sounds more natural imo